test-utils/src/main/java/com/revenuecat/purchases/utils/productStubs.kt

```diff
@@ -38,6 +38,32 @@ fun stubStoreProduct(
     override fun writeToParcel(dest: Parcel?, flags: Int) {}
 }
 
+@SuppressWarnings("EmptyFunctionBlock")
+fun stubINAPPStoreProduct(
+    productId: String
+): StoreProduct = object : StoreProduct {
+    override val productId: String
+        get() = productId
+    override val type: ProductType
+        get() = ProductType.INAPP
+    override val oneTimeProductPrice: Price?
+        get() = null
+    override val title: String
+        get() = ""
+    override val description: String
+        get() = ""
+    override val subscriptionPeriod: String?
+        get() = null
+    override val purchaseOptions: List<PurchaseOption>
+        get() = listOf(stubPurchaseOption(productId, emptyList()))
```

Comment:
Asuming INAPPs will have an empty list of pricing phases and a purchase option with an ID matching the product ID

Reply:
Yeah, part of what we need to discuss in the API for INAPPs... But I don't want to block this, so I think this is ok for now.